ohttp/Cargo.toml

```diff
@@ -16,6 +16,7 @@ nss = ["bindgen"]
 pq = ["hpke-pq"]
 rust-hpke = ["rand", "aead", "aes-gcm", "chacha20poly1305", "hkdf", "sha2", "hpke"]
 gecko = ["mozbuild"]
+app-svc = []
```

Comment:
I don't know why I didn't do this earlier, but it seems like both gecko and app-svc need to depend on nss.